src/eos-util.c

```diff
@@ -421,3 +452,142 @@ eos_updater_get_timestamp_from_branch_file_keyfile (GKeyFile *branch_file,
   *out_timestamp = g_steal_pointer (&timestamp);
   return TRUE;
 }
+
+gchar *
+eos_updater_dup_envvar_or (const gchar *envvar,
+                           const gchar *default_value)
+{
+  const gchar *value = g_getenv (envvar);
+
+  if (value != NULL)
+    return g_strdup (value);
+
+  return g_strdup (default_value);
```

Comment:
This function is getting copy pasted a lot. Can it go into `eos-updater-util.h`?